README.md

```diff
@@ -268,13 +234,13 @@ I would like to thank...
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[Question_1]**
+- **[Did you understand the Git Hub Process Flow]**
 
-  - [Answer_1]
+  - [Yes and will improve as I gain more experience working with Git Hub]
 
-- **[Question_2]**
+- **[Do you understand Git Process Flow]**
 
-  - [Answer_2]
+  - [Yes and will improve as I gain more experience working with Git]
```

Comment:
- [ ]  suggested by the previous Reviewer, Kindly provide a valid link for your project's License we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.